Bluetooth: tests/bsim/bluetooth/ll/bis/ accesses controller directly to update the channel map

**Describe the bug**
tests/bsim/bluetooth/ll/bis/src/main.c (L444) accesses directly the controller to update the channel 
map by calling `ll_chm_update` instead of going thru the HCI interface.
So this test cannot be built in the application core in split stack builds.

**To Reproduce**
N/A

**Expected behavior**
BT application level tests can be built also in split builds, i.e. they do not call into the BT controller directly.

**Impact**
The test cannot be used in split stack builds

**Logs and console output**
```
/usr/bin/ld: zephyr/build/bis/zephyr/zephyr.elf.loc_cpusw.o: in function `test_iso_main':
tests/bsim/bluetooth/ll/bis/src/main.c:444: undefined reference to `ll_chm_update'
collect2: error: ld returned 1 exit status
```

**Environment (please complete the following information):**
 - OS: Ubuntu 22.04
 - Toolchain (e.g Zephyr SDK, ...): Host gcc 11 or 12
 - Commit SHA or Version used: Main as of now baad781195ac1777bd5a9d9b61c7beaf4913aa2b

--- a/tests/bsim/bluetooth/ll/bis/src/main.c
+++ b/tests/bsim/bluetooth/ll/bis/src/main.c
@@ -441,7 +441,7 @@ static void test_iso_main(void)
 	k_sleep(K_MSEC(2500));
 
 	printk("Periodic Advertising and ISO Channel Map Update...");
-	err = ll_chm_update(chan_map);
+	err = bt_le_set_chan_map(chan_map);
 	if (err) {
 		FAIL("Channel Map Update failed.\n");
 	}

